Add `Yaz's Summer Cup 2017` tournament article and other prerequisites (#5856)
ppy/osu-wiki: wiki/Tournaments/YSC/2017/en.md

# Yaz's Summer Cup 2017

![YSC 2017 logo](img/logo.png)

> The **Yaz's Summer Cup 2017** (***YSC 2017***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) and sponsored by Ferocity Esports. It was the 1st installment of Yaz's Summer Cup.

Reviewer: ```suggestion
The **Yaz's Summer Cup 2017** (***YSC 2017***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) and sponsored by Ferocity Esports. It was the first instalment of Yaz's Summer Cup.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2017-06-17/2017-07-08 |
| Qualifiers/Round of 128 | 2017-07-21/2017-07-23 |
| Group stage drawings | 2017-07-24 |
| Group stage | 2017-07-28/2017-08-06 |
| Round of 32 | 2017-08-11/2017-08-13 |
| Round of 16 | 2017-08-18/2017-08-20 |
| Quarterfinals | 2017-08-25/2017-08-27 |
| Semifinals | 2017-09-01/2017-09-03 |
| Finals | 2017-09-09/2017-09-10 |
| Grand Finals | 2017-09-30 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | $60, profile badge, 6 months of osu!supporter tag |
> | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | $35, 4 months of osu!supporter tag |
> | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | $20, 2 months of osu!supporter tag |
> | *4th place* | $10, 1 month of osu!supporter tag |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | $60, profile badge, 6 months of osu!supporter |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | $35, 4 months of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | $20, 2 months of osu!supporter |
| *4th place* | $10, 1 month of osu!supporter |
```

![](img/badge.png "YSC 2017 winner badge")

## Organisation

> Yaz's Summer Cup 2017 was run by various community members by distributing the multitude of tasks into various fields of responsibility.

Reviewer: ```suggestion
Yaz's Summer Cup 2017 was run by various community members.
```
can use this version, we deprecated the long one pretty much

| Group N | ![][flag_DE] **[Firstus](https://osu.ppy.sh/users/1856829)**, ![][flag_AU] **[Lunirs](https://osu.ppy.sh/users/2118945)**, ![][flag_GB] [Karthy](https://osu.ppy.sh/users/4196808), ![][flag_AU] [Monk Gyatso](https://osu.ppy.sh/users/4012086) |
| Group O | ![][flag_US] **[Monko2k](https://osu.ppy.sh/users/4852013)**, ![][flag_US] **[Smoothie World](https://osu.ppy.sh/users/1855540)**, ![][flag_NL] [Timpai](https://osu.ppy.sh/users/5054798), ![][flag_RU] [ncuh](https://osu.ppy.sh/users/2891734) |
| Group P | ![][flag_ES] **[Betwin](https://osu.ppy.sh/users/1968481)**, ![][flag_GB] **[Helix](https://osu.ppy.sh/users/3322597)**, ![][flag_GB] [TopSp1n](https://osu.ppy.sh/users/7657761), ![][flag_EG] [KupcaH](https://osu.ppy.sh/users/7844575) |

See the [participants sheet](https://docs.google.com/spreadsheets/d/1cO_DR2ffyp0HuGtlHUHEV0MqQR-I0l7AgkQdygjjNmw/pubhtml?gridlines=false "Google Sheets") for a list of all 128 initial participants.

## Mappools

### Grand Finals

> - Nomod

Reviewer: ```suggestion
- NoMod
```

  - [airportexpress feat.Itsuneko - BIRTH (Chloe) [Insane]](https://osu.ppy.sh/beatmapsets/175241#osu/422762)
- DoubleTime
  - [Utagumi Setsugekka - Yumemi Sunrise (Zweib) [Insane]](https://osu.ppy.sh/beatmapsets/132312#osu/338862)
  - [Nightwish - She Is My Sin (liangv587) [Insane]](https://osu.ppy.sh/beatmapsets/233875#osu/542640)
  - [sana - cantabile (Zero\__wind) [Skyline]](https://osu.ppy.sh/beatmapsets/451910#osu/969420)
  - [Refresh - Summertime (ekowraith remix) (AndreasHD) [Insane]](https://osu.ppy.sh/beatmapsets/30581#osu/100880)
- FreeMod
  - [Ling tosite sigure - KIMITOOKU (Milan-) [pishi's Extra]](https://osu.ppy.sh/beatmapsets/458931#osu/988684)
  - [100-200-400 - Fascination MAXX (Mystearica) [28 KEY]](https://osu.ppy.sh/beatmapsets/15650#osu/56549)
  - [C-SHOW - Workaholic (ktgster) [Extra]](https://osu.ppy.sh/beatmapsets/103470#osu/272645)
  - [Falcom Sound Team jdk - RELEASE OF THE FAR WEST OCEAN (CSY the corrupt) [Corruption]](https://osu.ppy.sh/beatmapsets/77504#osu/217493)
- Tiebreaker
  - **[BlackYooh vs. siromaru - BLACK or WHITE? (Akali) [GRANDMASTER]](https://osu.ppy.sh/beatmapsets/463454#osu/992389)**

### Finals

> - Nomod

Reviewer: ```suggestion
- NoMod
```

  - [Igorrr - Tout Petit Moineau (fergas) [Extra]](https://osu.ppy.sh/beatmapsets/212180#osu/517308)
- DoubleTime
  - [REOL - ChiruChiru (Pho) [Scatter]](https://osu.ppy.sh/beatmapsets/476697#osu/1018250)
  - [GhostFinal - Grenade (Pokie) [Another]](https://osu.ppy.sh/beatmapsets/49507#osu/152819)
  - [The Prototypes - Slip Away (Secretpipe) [Collab Insane]](https://osu.ppy.sh/beatmapsets/518278#osu/1118651)
  - [Mitchie M - Suki ! Yuki ! Maji Magic (Natsu) [Snow]](https://osu.ppy.sh/beatmapsets/230442#osu/554701)
- FreeMod
  - [sakuzyo - Imprinting (EvilElvis) [Extreme]](https://osu.ppy.sh/beatmapsets/370843#osu/812665)
  - [Rita - Beyond the Eternity (Delis) [Desolation]](https://osu.ppy.sh/beatmapsets/630897#osu/1331749)
  - [Sakuzyo - Laplace (yf\_bmp) [AR8]](https://osu.ppy.sh/beatmapsets/427582#osu/923912)
  - [Tonight Alive - Little Lion Man (puxtu) [pishi's Extra]](https://osu.ppy.sh/beatmapsets/451311#osu/1046775)
- Tiebreaker
  - **[RIN - Pluto -Tenderness (My Angel Azusa) [Hades In The Heaven]](https://osu.ppy.sh/beatmapsets/573313#osu/1241443)**

### Semifinals

> - Nomod

Reviewer: ```suggestion
- NoMod
```

  - [My Chemical Romance - Helena (pishifat) [Kyshiro's Extra]](https://osu.ppy.sh/beatmapsets/320194#osu/712972)
- DoubleTime
  - [Ito Kanako - Skyclad no Kansokusha (Linada) [-Tochi's Insane]](https://osu.ppy.sh/beatmapsets/416129#osu/972844)
  - [nano - Now or Never (tutuhaha) [Never]](https://osu.ppy.sh/beatmapsets/51755#osu/158495)
  - [livetune feat. Hatsune Miku - Tell Your World (Colin Hou) [Insane]](https://osu.ppy.sh/beatmapsets/42110#osu/132608)
  - [Umetora - Ifuudoudou (Nymph) [Insane]](https://osu.ppy.sh/beatmapsets/84023#osu/231825)
- FreeMod
  - [ginkiha+kamome sano - Dawn of Asia (Rare) [INFINITE]](https://osu.ppy.sh/beatmapsets/122041#osu/596081)
  - [BABYMETAL - MEGITSUNE (Onisakura) [BluOxy's Challenging]](https://osu.ppy.sh/beatmapsets/110357#osu/343528)
  - [Sharlo - Hatsukoi no Ehon (Minakami Yuki) [Extra]](https://osu.ppy.sh/beatmapsets/332887#osu/736886)
  - [HujuniseikouyuuP - Talent Shredder (val0108) [0108 style]](https://osu.ppy.sh/beatmapsets/47710#osu/178966)
- Tiebreaker
  - **[Meramipop - Eiya [Imperishable Challengers] (-Tochi) [Eternal Night]](https://osu.ppy.sh/beatmapsets/289225#osu/652034)**

### Quarterfinals

> - Nomod

Reviewer: ```suggestion
- NoMod
```

  - [beatMARIO - Night of Knights (alacat) [Guy's Extra]](https://osu.ppy.sh/beatmapsets/352570#osu/786018)
- DoubleTime
  - [Lindsey Stirling - Senbonzakura (MrSergio) [Insane]](https://osu.ppy.sh/beatmapsets/277421#osu/720172)
  - [Kozato - Tsuki -Yue- (jonathanlfj) [Another]](https://osu.ppy.sh/beatmapsets/101123#osu/268080)
  - [Rin - Spring of Dreams (jyc-Binggan) [Lunatic]](https://osu.ppy.sh/beatmapsets/119739#osu/325938)
  - [MitiS - Open Window (feat. Anna Yvette) ([Luanny]) [Collab]](https://osu.ppy.sh/beatmapsets/100628#osu/266925)
- FreeMod
  - [Fall Out Boy - Alone Together (Krewella Remix) (Aia) [Extra]](https://osu.ppy.sh/beatmapsets/290995#osu/784411)
  - [Dreamcatcher - GOOD NIGHT (Natsu) [toybot's Sunset]](https://osu.ppy.sh/beatmapsets/603905#osu/1286416)
  - [jippusu - Mushikui Saikede Rhythm (Amamiya Yuko) [Skystar]](https://osu.ppy.sh/beatmapsets/87547#osu/239104)
  - [Giga-P - LUVORATORRRRRY! (Selisi) [HW's Extra]](https://osu.ppy.sh/beatmapsets/153206#osu/394808)
- Tiebreaker
  - **[Bullet For My Valentine - Tears Don't Fall (FCL) [The Poison]](https://osu.ppy.sh/beatmapsets/374922#osu/821273)**

### Round of 16

> - Nomod

Reviewer: ```suggestion
- NoMod
```

- DoubleTime
  - [zts - lastendconductor (EvilElvis) [Insane]](https://osu.ppy.sh/beatmapsets/333965#osu/738959)
  - [Flo Rida - Run ft. RedFoo of LMFAO (Speed Up Ver.) (ohad1881) [A Mystery's Insane]](https://osu.ppy.sh/beatmapsets/283731#osu/682966)
  - [Doxx & Feint - Mind In Motion (Secretpipe) [Insane]](https://osu.ppy.sh/beatmapsets/222565#osu/520270)
  - [Miyazono Kaori(CV.Taneda Risa) - My Truth\~Rondo Capriccioso (Setz) [April]](https://osu.ppy.sh/beatmapsets/274337#osu/634245)
- FreeMod
  - [bj.HaLo - Ende (galvenize) [Another]](https://osu.ppy.sh/beatmapsets/44035#osu/148716)
  - [Itou Kanako - Skyclad no Kansokusha -Remix- (Takos) [taka]](https://osu.ppy.sh/beatmapsets/33323#osu/110628)
  - [Aiobahn & Yunomi - Ginga Tetsudou no Penguin ft. nicamoq (Stripe.P Remix) (deetz) [Collab]](https://osu.ppy.sh/beatmapsets/442510#osu/951600)
  - [Ryu\* - Mind Mapping (kors k mix) (wring) [EXTRA]](https://osu.ppy.sh/beatmapsets/47199#osu/146583)
- Tiebreaker
  - **[Akira Complex - Odyssey (Au5 Remix) (ProfessionalBox) [Excursion]](https://osu.ppy.sh/beatmapsets/341007#osu/754143)**

### Round of 32

> - Nomod

Reviewer: ```suggestion
- NoMod
```

- HardRock
  - [Demetori - Emotional Skyscraper \~ World's End (Lybydose) [Lunatic]](https://osu.ppy.sh/beatmapsets/13204#osu/48985)
  - [nano - Nevereverland (Nyquill) [Insane]](https://osu.ppy.sh/beatmapsets/95533#osu/256499)
  - [Goose house - Hikaru nara (ProfessionalBox) [Synesthesia]](https://osu.ppy.sh/beatmapsets/440169#osu/947166)
- DoubleTime
  - [Ailee - I Will Show You (Norman) [Hinsvar's Insane]](https://osu.ppy.sh/beatmapsets/85732#osu/246194)
  - [Samantha Jade - Soldier (7th Heaven Remix) (Rakuen) [Flask's Insane]](https://osu.ppy.sh/beatmapsets/157234#osu/716691)
  - [Nightwish - She Is My Sin (NatsumeRin) [Insane]](https://osu.ppy.sh/beatmapsets/13445#osu/49748)
- FreeMod
  - [Masayoshi Minoshima - Flight of the Bamboo Cutter (a3272509123) [Lunatic]](https://osu.ppy.sh/beatmapsets/47754#osu/148000)
  - [Hikasa Yoko - Listen!! (CDFA) [Collab]](https://osu.ppy.sh/beatmapsets/24610#osu/83603)
  - [NegaRen - Stark Raving Mad (vipto) [Raving Mad]](https://osu.ppy.sh/beatmapsets/54618#osu/166350)
- Tiebreaker
  - **[Dark PHOENiX - Green-Eyed Jealousy (sjoy) [Envy]](https://osu.ppy.sh/beatmapsets/170913#osu/413597)**

> ### Group Stage

Reviewer: ```suggestion
### Group stage
```

> - Nomod

Reviewer: ```suggestion
- NoMod
```

- HardRock
  - [Pizuya's Cell x MyonMyon - Romantic Children (Frill) [Lunatic]](https://osu.ppy.sh/beatmapsets/18009#osu/68431)
  - [Yooh - LegenD. (Flask) [ExhausT.]](https://osu.ppy.sh/beatmapsets/396839#osu/863364)
  - [Yuyoyuppe - Reon (Irie Miyuki) [Insane]](https://osu.ppy.sh/beatmapsets/55980#osu/244117)
- DoubleTime
  - [Dirty Elegance - Leaves Of Autumn (Mystyk) [Insane]](https://osu.ppy.sh/beatmapsets/350245#osu/772216)
  - [Basement Jaxx (feat. Lisa Kekaula) - Good Luck (Lesjuh) [Hard]](https://osu.ppy.sh/beatmapsets/20650#osu/72075)
  - [DECO\*27 feat. topi - Light Lag (happy30) [Insane]](https://osu.ppy.sh/beatmapsets/130583#osu/329301)
- FreeMod
  - [Krewella - Come & Get It (Nightcore Mix) (Asphyxia) [Insane]](https://osu.ppy.sh/beatmapsets/125553#osu/348605)
  - [Traktion - Mission ASCII (galvenize) [Another]](https://osu.ppy.sh/beatmapsets/53249#osu/162405)
  - [Seiryu - Ultramarine (RLC) [Another]](https://osu.ppy.sh/beatmapsets/107377#osu/281389)
- Tiebreaker
  - **[Shoji Meguro - Kimi no Kioku (Aethral Remix) (Akali) [Remembrance]](https://osu.ppy.sh/beatmapsets/490662#osu/1045757)**

### Qualifiers / Round of 128

> - Nomod

Reviewer: ```suggestion
- NoMod
```

- HardRock
  - [Jun Maeda x yanaginagi - Killer Song (Yohanes) [Pandiar's Insane]](https://osu.ppy.sh/beatmapsets/454725#osu/1033305)
  - [Himeringo - Kodoku no Kakurenbo (Asahina Momoko) [Frey's Insane]](https://osu.ppy.sh/beatmapsets/319791#osu/804824)
  - [NOMA - LOUDER MACHINE (Skystar) [toybot's Insane]](https://osu.ppy.sh/beatmapsets/483606#osu/1035096)
- DoubleTime
  - [FELT - Hail Storm (Kite) [Challenging]](https://osu.ppy.sh/beatmapsets/122349#osu/312973)
  - [Toshimasa - Soshite Kyou mo Koumakan no Tobira wa Hiraku (xxheroxx) [Sallad's Lunatic]](https://osu.ppy.sh/beatmapsets/51558#osu/159851)
  - [ClariS - STEP (Rizia) [Nisestep]](https://osu.ppy.sh/beatmapsets/312851#osu/707636)
- FreeMod
  - [Demetori - Hoshi no Utsuwa \~ Casket of Star (happy30) [sjoy's Insane]](https://osu.ppy.sh/beatmapsets/139677#osu/1115583)
  - [MUZIK SERVANT feat. TAM - Upright Blue (Reiji-RJ) [Collab]](https://osu.ppy.sh/beatmapsets/56450#osu/170677)
  - [Tia - The Glory Days (Nara) [P A N's Insane]](https://osu.ppy.sh/beatmapsets/391953#osu/903567)
- Tiebreaker
  - **[Dancing Dolls - monochrome(Asterisk Makina Remix) (monstrata) [Scythe]](https://osu.ppy.sh/beatmapsets/349445#osu/770576)**

> ## Match Results

Reviewer: ```suggestion
## Match results
```

#### Winners Finals

Sunday, 10 September 2017:

| Player 1 | | | Player 2 | Match info |
| --: | :-: | :-: | :-- | :-- |
| **Vaxei** ![][flag_US] | **6** | 0 | ![][flag_US] Toy |  |

#### Losers Bracket Round 6

Friday, 8 September 2017:

| Player 1 | | | Player 2 | Match info |
| --: | :-: | :-: | :-- | :-- |
| **Cookiezi** ![][flag_KR] | **6** | 5 | ![][flag_US] he rack |  |
| **Azer** ![][flag_CA] | **6** | 1 | ![][flag_RO] Rohulk |  |

#### Losers Semifinals

Saturday, 9 September 2017:

| Player 1 | | | Player 2 | Match info |
| --: | :-: | :-: | :-- | :-- |
| **Cookiezi** ![][flag_KR] | **6** | 1 | ![][flag_CA] Azer |  |

#### Losers Finals

Saturday, 16 September 2017:

| Player 1 | | | Player 2 | Match info |
| --: | :-: | :-: | :-- | :-- |
| Cookiezi ![][flag_KR] | 5 | **6** | ![][flag_US] **Toy** |  |

### Semifinals

#### Winners Semifinals

Saturday, 2 September 2017:

| Player 1 | | | Player 2 | Match info |
| --: | :-: | :-: | :-- | :-- |
| Rohulk ![][flag_RO] | 4 | **6** | ![][flag_US] **Vaxei** |  |

Sunday, 3 September 2017:

| Player 1 | | | Player 2 | Match info |
| --: | :-: | :-: | :-- | :-- |
| **Toy** ![][flag_US] | **6** | 2 | ![][flag_US] he rack |  |

> #### Losers Bracker Round 4

Reviewer 2: do these intend to say "bracket"

Author: Yes

| Player 1 | | | Player 2 | Match info |
| --: | :-: | :-: | :-- | :-- |
| **Electovoid** ![][flag_GB] | **3** | 0 | ![][flag_SI] Hranolka |  |
| **Monko2k** ![][flag_US] | **3** | 0 | ![][flag_GB] SteffoST |  |

Thursday, 27 July 2017:

| Player 1 | | | Player 2 | Match info |
| --: | :-: | :-: | :-- | :-- |
| **Smoothie World** ![][flag_US] | **3** | 2 | ![][flag_LT] Damach |  |
| **Wombraider___9** ![][flag_US] | **1** | 0 | ![][flag_GB] Nega | *forfeit* |
| **idke** ![][flag_US] | **1** | 0 | ![][flag_NO] YokesPai | <!-- TODO check with yaz what she remember about this match --> |

## Ruleset

- This tournament is a osu!, 1v1, international tournament.
- This tournament will consist of 128 players.
- Matches will be head to head, with score v2 as the scoring system used.
- There will be no rank limit on signups, however if we reach 128 signups we will take the top 128 players in terms of rank.
- Ranks will be taken and locked when registrations have concluded
- Anyone who does sign up must be able to play maps with a difficulty of at least 4.7*.
- Starting from groups, the tournament will be double elimination. Qualifications will be the only thing that is single elimination.

> - All staff members are allowed to sign up for the tournament. The only exception are the host, co-hosts, and mappool selectors/testers.

Reviewer 2: ```suggestion
- All staff members are allowed to sign up for the tournament. The only exceptions are the host, co-hosts, and mappool selectors/testers.
```

> - All players must treat staff and other participants with a high level of respect, any kind of mistreatment in a match or on the discord (this includes extreme toxicity, racism, homophobia) will result in ban and will be blacklisted from participating in future installments of this tournament.

Reviewer 2: ```suggestion
- All players must treat staff and other participants with a high level of respect, any kind of mistreatment in a match or on the Discord server (this includes extreme toxicity, racism, homophobia) will result in a ban and will be blacklisted from participating in future installments of this tournament.
```

- Where rules do not prevail, common sense shall. The administration has explicit discretion to apply their judgement on this as they see fit.

> - Each stage will be held on a friday/saturday/sunday. Schedules and mappools will be released on the monday/tuesday before that stage is due to happen, to allow for reschedules.

Reviewer 2: ```suggestion
- Each stage will be held on a Friday/Saturday/Sunday. Schedules and mappools will be released on the Monday/Tuesday before that stage is due to happen to allow for reschedules.
```
also consider removing pause-comma; nitpick

- Each stage will be held on a Friday/Saturday/Sunday. Schedules and mappools will be released on the Monday/Tuesday before that stage is due to happen, to allow for reschedules.
- Reschedules will be allowed if both players agree on a time and provide proof of the agreed reschedule to hosts/co-hosts/scheduling team.
- Please be aware that reschedules must happen at least 24 hours before the match is due to happen. Any time after this will not be considered.
- In order for your registration to count, you must fill out the form provided, join the Discord server and change your Discord username to your osu! username.

> ### General Rules

Reviewer: ```suggestion
### General rules
```

- Referees will create the lobby and invite players 10 minutes before the match is due to start.

> - If a player fails to show up after their match is due to start, a 10 minute timer will be started.

Reviewer 3: redundant (combined with below bullet point)

```suggestion
```

> - Any player that fails to show up after the 10 minute timer will forfeit the match and a default win will be awarded to the other player.

Reviewer 3: ```suggestion
- Any player that fails to show up 10 minutes after a match is due to start will forfeit the match and a win by default will be awarded to the other player.
```

- Disconnects during the first 30 seconds of the map will result in the map being replayed. Any disconnections after this will result in a forfeit of that map.
- Players are only allowed to disconnect once per match. Anything after this will result in the match being forfeited to the other player.
- Any attempts to stall the match, show toxicity or spam during the match will result in a warning. Failure to comply with this rule again will result in a point being awarded to another player.
- In a FreeMod pick, both players must choose a mod (HD/HR/EZ) or a combination of these mods.
- The score of a player who failed during a map but managed to revive before the end will be counted as if it were a pass.
- The score of a player who is in failed state when the map has finished will be counted as 0.
- If both players fail, the highest failed score will win the map.
- If there is a tie, the tiebreaker will be played to determine the winner of that match (mods are available during the tiebreaker).
- At the end of the match, you are free to leave the lobby.

> ### Qualifiers and Group Stage

Reviewer: ```suggestion
### Qualifiers and Group stage
```

- There will be no warmups and one ban per player.
- Players will roll once. The winner of that roll will ban and pick first.

> - In groups, there will be one top, one high, one mid and one low seeded player per group, this will be drawn randomly.

Reviewer 3: ```suggestion
- In groups, there will be one top, one high, one mid and one low seeded player per group, which will be drawn randomly.
```

- During these stages, players are not allowed to choose from the same mod bracket (HD/HR/DT/FreeMod) twice in a row. This excludes the NoMod bracket pool.
- The top 2 players from each group will make it through to the knockout stage.

> ### Knockout Stages

Reviewer: ```suggestion
### Knockout stages
```
